python/Ganga/test/GPI/FileTests/TestMassStorageClient.py

```diff
@@ -30,13 +32,19 @@ def setUp(self):
         extra_opts=[('PollThread', 'autostart', 'False'),
                     ('Local', 'remove_workdir', 'False'),
                     ('TestingFramework', 'AutoCleanup', 'False'),
-                    ('Output', 'MassStorageFile', TestMassStorageClient.MassStorageTestConfig),
+                    ('Output', TestMassStorageClient.fileName, TestMassStorageClient.MassStorageTestConfig),
                     ('Output', 'FailJobIfNoOutputMatched', 'True')]
         super(TestMassStorageClient, self).setUp(extra_opts=extra_opts)
 
+    @classmethod
+    def getFileObject(cls):
+        """ Return an instance of the file we're wanting to test """
+        import Ganga.GPI as gpi
+        return getattr(gpi, cls.fileName)
```

Comment:
I was wanting something sane, small and quick. I was not wanting to go messing with the Plugin system's internals here. This required things working and I need the proxied version of the class in that case so would likely have to call the proxy layer in addition.
I just didn't want to do in about 7-8 lines what can be done in 2.
I was toying with a clever import method that uses the strings but settled for this which works and is simple.